alma/alma-installments-prestashop
# [insurance-refacto] handle savecart module with insurance

### Reason for change



[Linear task](https://linear.app/almapay/issue/ECOM-1691/[🧩-ps]-fix-compatibility-with-a-savecart-module)

### Code changes


On save cart (if curent cart id != new cart id), on the hook saveCart, we duplicate the cart

### How to test

_As a reviewer, you are encouraged to test the PR locally._


Install a module for save cart (eg.: opartsavecart)
Save a cart with insurance and check if the insurance design product is displayed

### Checklist for authors and reviewers




### Non applicable
